cli/getting-started/index.md

```diff
@@ -406,7 +406,7 @@ experience. Terramate Cloud is free for individual use, with features available
 To start using the cloud, you need to sign up for a free cloud account and create an organization.
 
 - Sign in and Sign up to [cloud.terramate.io](https://cloud.terramate.io) and
-- Create your Organization
+- Create your Organization. You can edit GitHub Trust [settings](../../cloud/organization/settings.md#general-settings) to establish a trust relationship with GitHub Actions and specify whether to sync the entire organization or a specific repository with Terramate Cloud.
```

Comment:
I think the wording needs to be improved to make clear that this is **required** to sync data from Github or Gitlab.

Reply:
Can we also add:
- the expected value in the trust field
- an example of an account level trust 
- an example of a repo level trust

Reply:
the expected value in the Trust field is documented already here: https://terramate.io/docs/cloud/organization/settings#edit-organization-settings-as-an-admin

<img width="717" alt="Bildschirmfoto 2024-09-25 um 23 09 30" src="https://github.com/user-attachments/assets/3fdb0189-9918-4c46-8725-598d0b4d76b0">